rpc/provider.go

```diff
@@ -49,7 +49,6 @@ type api interface {
 	TransactionByBlockIdAndIndex(ctx context.Context, blockID BlockID, index uint64) (Transaction, error)
 	TransactionByHash(ctx context.Context, hash *felt.Felt) (Transaction, error)
 	TransactionReceipt(ctx context.Context, transactionHash *felt.Felt) (TransactionReceipt, error)
-	TransactionTrace(ctx context.Context, transactionHash *felt.Felt) (TxnTrace, error)
```

Comment:
Why is this being removed?

Reply:
This is weird. It should have been included in the "merge main" commit but seemed to have been dropped??